pkg/operator/sync.go

```diff
@@ -700,6 +701,9 @@ func newContainers(config *OperatorConfig, features map[string]bool) []corev1.Co
 						FieldPath: "spec.nodeName",
 					},
 				},
+			}, corev1.EnvVar{
+				Name:  "OPERATOR_IMAGE_VERSION",
```

Comment:
Curious why you've chosen this name over `RELEASE_VERSION`?

Reply:
Here's[1] the function I found in library-go for getting operator version, which pulls from the environment variable `OPERATOR_IMAGE_VERSION`

[1] https://github.com/openshift/library-go/blob/master/pkg/operator/status/version.go#L76

Reply:
and I'm using it here: https://github.com/openshift/machine-api-provider-azure/commit/b317300432a2b37ada7fcd3ece06fba75afd216f#diff-435eecb1d2af40ee96747f88ab71eb2758da989c92f76dcb1f714fc1b300c633R135

Reply:
I think we should be sticking to `RELEASE_VERSION` since we don't distinguish between operator and operand release versions within the MAPI ecosystem, everything should be the same release version